source/web-stream.js

```diff
@@ -0,0 +1,13 @@
+export const ponyfill = {};
+
+const {prototype} = ReadableStream;
+
+// Use this library as a ponyfill instead of a polyfill.
+// I.e. avoid modifying global variables.
+// We can remove this once https://github.com/Sec-ant/readable-stream/issues/2 is fixed
+if (prototype[Symbol.asyncIterator] === undefined && prototype.values === undefined) {
```

Comment:
This condition is always `false` on Node, Firefox and Chrome >124. On those platforms, this file is a noop.